### DISCUSSION TITLE
Request: Markdown icon

### DISCUSSION THREAD

**Discussion by nickjj** [tailwindlabs/heroicons#363] [Ideas]
Here's references to the official outline and solid variants (licensed to the public domain):

- Outline: https://github.com/dcurtis/markdown-mark/blob/master/svg/markdown-mark.svg
- Solid: https://github.com/dcurtis/markdown-mark/blob/master/svg/markdown-mark-solid.svg

You might be wondering, *"if we start adding project / technology specific icons where do we draw the line? There's hundreds of them!"* and I agree but a Markdown icon is a bit interesting because it plays an important UX role on any site where you accept user input and support Markdown.

For example look at GitHub's issue comment box. It displays a Markdown icon near the bottom. Would be nice to have.

**Comment by bigint** [comment]
+1

**Comment by bigint** [comment]
I'm using this for now 😓 

![image](https://user-images.githubusercontent.com/69431456/123599192-fa94d180-d812-11eb-96d5-c8a85b1919d3.png)
